Boa.Constrictor/WebDriver/Tasks/Drag.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Boa.Constrictor.Screenplay;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+
+namespace Boa.Constrictor.WebDriver
+{
+    /// <summary>
+    /// Drags the mouse from one element to another
+    /// </summary>
+    public class Drag : AbstractWebLocatorTask
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Private constructor.
+        /// (Use static builder methods to construct.)
+        /// </summary>
+        /// <param name="source">The source web element locator</param>
+        /// <param name="target">The target web element locator</param>
+        private Drag(IWebLocator source, IWebLocator target) : base(source) 
+        {
+            Target = target;
+        }
+
+        #endregion
+
+        #region Properties
+        private IWebLocator Target { get; set; }
```

Comment:
Despite being `private`, please add `///` comments.

Reply:
Also, there's no reason to make this private. Let's make it public. The `Locator` property is `public`.

Reply:
updated as requested